alembic/env.py

```diff
@@ -30,11 +32,12 @@
 # ... etc.
 
 
-def do_run_migrations(connection):
+def do_run_migrations(connection: engine.base.Connection) -> None:
     """Run a migration given an async connection.
 
     A helper function used by run_migrations_online.
     """
+    print("connection is an ", type(connection))
```

Comment:
Do you mean this print to be here (and why isn't it a log message if it is meant to be here)

Reply:
Good catch. It was meant to be temporary to figure out mypy annotation. I'll remove it.